python/pyspark/worker.py

```diff
@@ -57,6 +57,13 @@ def main(infile, outfile):
         if split_index == -1:  # for unit tests
             exit(-1)
 
+        version = read_int(infile)
+        version = (version // 10, version % 10)
```

Comment:
What does `//` do? Scala shell seems to indicate it's a typo:

    scala> 100 // 10
    res0: Int = 100